docs/detections/rules-ui-monitor.asciidoc

```diff
@@ -23,8 +23,6 @@ image::images/monitor-table.png[]
 
 On the *Rule Monitoring* tab, you can <<sort-filter-rules, sort and filter rules>> just like you can on the *Rules* tab. 
 
-TIP: By default, you can only sort by the *Rule* and *Enabled* columns on this tab. To sort by any other column, turn on the *Advanced sorting* toggle above the table. If you experience performance issues when working with the table, you can turn this setting off.
```

Comment:
I'm debating including the tip that you can sort by clicking a header. I feel like it's common functionality that most users would expect but I also don't want make assumptions that force users to spend extra time figuring out the UI on their own.  

If we were to add the tip back in, it'd just be:

To sort the rules list, click any column header. To sort in descending order, click the column header again.

Reply:
Sounds good, added the tip back 👍